knowledge-base/common-grpc-services-in-blazor.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This already exists at https://docs.telerik.com/blazor-ui/knowledge-base/grid-grpc

Reply:
Removed